src/components/make_component.tsx

```diff
@@ -11,15 +11,16 @@ import { useWidgetRenderer, StylingProps } from "./renderer"
 export type Props = SsoUrlProps &
   StylingProps &
   WidgetConfigurationProps &
-  WidgetPostMessageCallbackProps<string>
+  WidgetPostMessageCallbackProps<string> &
+  JSX.IntrinsicAttributes
```

Comment:
Needed after migrating from React v17 to v18 (SDK uses 17, example app uses 18). It's possible we'll be able to revert this change after upgrading the SDK.